edge-modules/api-proxy-module/templates/nginx_default_config.conf

```diff
@@ -68,6 +68,24 @@ http {
         }
         #endif_tag ${BLOB_UPLOAD_ROUTE_ADDRESS}        
 
+        #if_tag ${CONNECTED_ACR_ROUTE_ADDRESS}
```

Comment:
@toddysm, can you review the name `CONNECTED_ACR_ROUTE_ADDRESS` ? The customer will use it to configure the connected registry container endpoint.